mq/esworker_pubsub.py

```diff
@@ -0,0 +1,188 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+# Copyright (c) 2017 Mozilla Corporation
+
+import json
+import sys
+import os
+import socket
+import time
+from configlib import getConfig, OptionParser
+from datetime import datetime
+from mozdef_util.utilities.toUTC import toUTC
+from mozdef_util.utilities.logger import logger, initLogger
+from mozdef_util.elasticsearch_client import (
+    ElasticsearchClient,
+    ElasticsearchBadServer,
+    ElasticsearchInvalidIndex,
+    ElasticsearchException,
+)
+from google.cloud import pubsub
+
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), "../"))
```

Comment:
We're trying to get away from modifying sys.path, so this can be replaced with `from lib.plugins import sendEventToPlugins, registerPlugins`